ui/src/pages/SearchPage.tsx

```diff
@@ -421,6 +421,8 @@ type MatchDateT = "null" | "lt" | "exact" | "gt" | "bt";
 // components API does not include null matcher
 type MatchDateT = "lt" | "gt";
 
+type MatchLicenseT = "null" | "icontains" | "exact";
```

Comment:
feature: changes to add support for component license none|any values start here